zeebe/gateway-rest/src/main/java/io/camunda/zeebe/gateway/rest/SearchQueryRequestMapper.java

```diff
@@ -90,6 +94,21 @@ public final class SearchQueryRequestMapper {
 
   private SearchQueryRequestMapper() {}
 
+  public static Either<ProblemDetail, UsageMetricsQuery> toUsageMetricsQuery(
+      final UsageMetricsSearchQueryRequest request) {
+    if (request == null || (request.getStartTime() == null && request.getEndTime() == null)) {
```

Comment:
🔧 I guess all of them need to be defined, right?
```suggestion
    if (request == null || request.getStartTime() == null || request.getEndTime() == null) {
```

Reply:
Adjusting this accordingly